.github/workflows/ci.yml

```diff
@@ -6,8 +6,10 @@ env:
 
 jobs:
   linting:
-    runs-on: ubuntu-latest
     name: Linting (${{matrix.elixir}}/${{matrix.otp}})
+    runs-on: ubuntu-latest
+
+    if: github.event_name == 'push' || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == true)
```

Comment:
Good find on this! Would you mind submitting this as a separate PR so we can use it immediately?

Reply:
Yep, no problem